scripts/utils/trading_strategy_helpers.js

```diff
@@ -429,30 +429,35 @@ withdrawal of or to withdraw the desired funds
   const buildTransferApproveDepositFromOrders = async function(
     masterAddress,
     bracketAddresses,
+    targetTokenAddress,
     stableTokenAddress,
+    lowestLimit,
+    highestLimit,
+    targetPrice,
     investmentStableToken,
-    targetTokenAddress,
     investmentTargetToken,
     storeDepositsAsFile = false
   ) {
     const fleetSize = bracketAddresses.length
-    assert(fleetSize % 2 == 0, "Fleet size must be a even number")
+    const stepSizeAsMultiplier = Math.pow(highestLimit / lowestLimit, 1 / bracketAddresses.length)
+    const bracketIndexAtTargetPrice = Math.floor(Math.log(targetPrice / lowestLimit) / Math.log(stepSizeAsMultiplier))
+    assert(bracketIndexAtTargetPrice <= fleetSize, "TargetPrice is not between the bounds")
```

Comment:
Shouldn't the assertion use just `<` instead? I'd also assert that `bracketIndexAtTargetPrice>=0`.

Reply:
It arguably makes sense that the index is outside this range. This is the case if you want to anticipate a movement of the price and want to create your strategy outside of a reasonable range. For example, let's say that the current ETH price is 100 but you believe the price is 200 and want to prepare your brackets between 150 and 250. In this case it's reasonable that `bracketIndexAtTargetPrice` is outside the range, and all brackets are filled with only ETH. That's because `targetPrice` would not really be your "target" but rather the current price. It doesn't make sense to talk about a "target", since everything is defined around min and max of a range. Does this make sense? This would change the PR a bit.